sips/sip-16/sip-016-token-metadata.md

```diff
@@ -0,0 +1,479 @@
+        },
+        "localization": {
+            "type": "object",
+            "required": ["uri", "default", "locales"],
+            "properties": {
+                "uri": {
+                    "type": "string",
+                    "description": "The URI pattern to fetch localized data from. This URI should contain the substring `{locale}` which will be replaced with the appropriate locale value before sending the request. See section about localization for more rules"
+                },
+                "default": {
+                    "type": "string",
+                    "description": "The locale of the default data within the base JSON"
+                },
+                "locales": {
+                    "type": "array",
+                    "description": "The list of locales for which data is available. These locales should conform to those defined in the Unicode Common Locale Data Repository (http://cldr.unicode.org/)."
+                }
+            }
+        },
+        "image_data": {
+            "type": "string",
+            "description": "Raw SVG image data. Deprecated. Use `properties.image_data`."
+        },
+        "external_url": {
+            "type": "string",
+            "description": "Url to view the item on a 3rd party web site. Deprecated. Use `properties.external_url`."
+        },
+        "animation_url": {
+            "type": "string",
+            "description": "URL to a multi-media attachment for the item. Deprecated. Use `properties.animation_url`."
+        },
+    }
+}
+```
+
+The length of string values is not restricted. Nowadays, clients should be smart
+enough to deal with values of different lengths. Note, that the [sitemap
+protocol](https://www.sitemaps.org/protocol.html) and many search engines
+support only URLs with less than 2048 characters.
+
+### Example
+
+token101.json
+
+```
+{
+  "sip": 16,
+  "name": "Foo #101",
+  "image": "ipfs://somerandomecid",
+  "attributes": [
+     {
+      "trait_type": "hair",
+      "value": "red",
+    },
+    {
+      "trait_type": "strength",
+      "display_type": "number",
+      "value": 99,
+    },
+  ],
+  "properties": {
+      "collection":  "Foo Collection",
+      "total_supply":  "10000"
+  },
+  "localization": {
+      "uri": "ipfs://somerandomcid/{locale}.json",
+      "default": "en",
+      "locales": ["en", "pt-BR", "de"]
+  }
+}
+```
+
+de.json
+
+```
+{
+    "sip": 16,
+    "attributes: [
+        {
+          "trait_type": "Haare",
+          "value": "rot",
+        },
+        {
+          "trait_type": "Stärke",
+          "display_type": "number",
+          "value": 99,
+        },
+    ]
+}
+```
+
+pt-BR.json
+
+```
+{
+    "sip": 16,
+    "attributes: [
+        {
+          "trait_type": "cabelos",
+          "value": "vermelho",
+        },
+        {
+          "trait_type": "amido",
```

Comment:
if this was supposed to be strength in brazilian portuguese the correct would be "força" 
amido is starch

Reply:
Thank you. Fixed!